concrete-core-fixture/src/fixture/glwe_ciphertext_ggsw_ciphertext_discarding_external_product.rs

```diff
@@ -0,0 +1,184 @@
+    for GlweCiphertextGgswCiphertextDiscardingExternalProductFixture
+where
+    Precision: IntegerPrecision,
+    Engine:
+        GlweCiphertextGgswCiphertextDiscardingExternalProductEngine<GlweInput, GgswInput, Output>,
+    GlweInput: GlweCiphertextEntity,
+    GgswInput: GgswCiphertextEntity<KeyDistribution = GlweInput::KeyDistribution>,
+    Output: GlweCiphertextEntity<KeyDistribution = GlweInput::KeyDistribution>,
+    SecretKey: GlweSecretKeyEntity<KeyDistribution = GlweInput::KeyDistribution>,
+    PlaintextVector: PlaintextVectorEntity,
+    Maker: SynthesizesGlweSecretKey<Precision, SecretKey>
+        + SynthesizesGlweCiphertext<Precision, GlweInput>
+        + SynthesizesGlweCiphertext<Precision, Output>
+        + SynthesizesGgswCiphertext<Precision, GgswInput>
+        + SynthesizesPlaintextVector<Precision, PlaintextVector>,
+{
+    type Parameters = GlweCiphertextGgswCiphertextDiscardingExternalProductParameters;
+    type Outcome = (Vec<Precision::Raw>, Vec<Precision::Raw>);
+    type RepetitionPrototypes = (
+        <Maker as PrototypesGlweSecretKey<Precision, GlweInput::KeyDistribution>>::GlweSecretKeyProto,
+    );
+    type SamplePrototypes = (
+        <Maker as PrototypesPlaintextVector<Precision>>::PlaintextVectorProto,
+        Vec<Precision::Raw>,
+    );
+    type PreExecutionContext = (GlweInput, GgswInput, Output);
+    type PostExecutionContext = Output;
+
+    fn generate_parameters_iterator() -> Box<dyn Iterator<Item = Self::Parameters>> {
+        Box::new(
+            vec![
+                GlweCiphertextGgswCiphertextDiscardingExternalProductParameters {
+                    noise: Variance(0.00000001),
+                    glwe_dimension: GlweDimension(200),
+                    polynomial_size: PolynomialSize(200),
+                    decomposition_base_log: DecompositionBaseLog(8),
+                    decomposition_level_count: DecompositionLevelCount(4),
+                },
+            ]
+            .into_iter(),
+        )
+    }
+
+    fn generate_random_repetition_prototypes(
+        parameters: &Self::Parameters,
+        maker: &mut Maker,
+    ) -> Self::RepetitionPrototypes {
+        let proto_secret_key =
+            maker.new_glwe_secret_key(parameters.glwe_dimension, parameters.polynomial_size);
+        (proto_secret_key,)
+    }
+
+    fn generate_random_sample_prototypes(
+        parameters: &Self::Parameters,
+        maker: &mut Maker,
+        _repetition_proto: &Self::RepetitionPrototypes,
+    ) -> Self::SamplePrototypes {
+        let raw_plaintext_vector = Precision::Raw::uniform_vec(parameters.polynomial_size.0);
+        let proto_plaintext_vector =
+            maker.transform_raw_vec_to_plaintext_vector(&raw_plaintext_vector);
+        (proto_plaintext_vector, raw_plaintext_vector)
+    }
+
+    fn check_sample_outcomes(parameters: &Self::Parameters, outputs: &[Self::Outcome]) -> bool {
```

Comment:
Sorry @orelcosseron but this has changed in a recent commit on master :cry: You'll have to implement a `compute_criteria` function and a `verify` function instead.